### PR TITLE
LPS-190314 Apply CompanyScoped interface for ObjectEntryManager as well

### PR DESCRIPTION
**Pull request by MarinhoFeliphe**

### CODE REVIEW

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/FunctionObjectActionExecutorImpl.java [hunk 1]
```diff
@@ -62,32 +66,30 @@ public void execute(
...
+	public String getKey() {
+		return _key;
 	}
 
 	@Activate
 	protected void activate(Map<String, Object> properties) throws Exception {
 		_allowedObjectDefinitionNames = StringUtil.asList(
 			properties.get("allowedObjectDefinitionNames"));
+
+		if (_allowedObjectDefinitionNames.isEmpty()) {
```
Comment (guilhermedcamacho) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/FunctionObjectActionExecutorImpl.java:88]: Hey @MarinhoFeliphe , do we really need to add the "ALL_OBJECT_DEFINITIONS" in allowedObjectDefinitionNames when it's empty? To simplify I would keep it empty and assume that when it's empty there is no configuration to specific objects, so it should work for all objects.

File: modules/apps/object/object-api/src/main/java/com/liferay/object/scope/ObjectDefinitionsScoped.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+	public List<String> getAllowedObjectDefinitionNames();
+
+	public default boolean isAllowedObjectDefinition(
+		String objectDefinitionName) {
+
+		List<String> allowedObjectDefinitionNames =
+			getAllowedObjectDefinitionNames();
+
+		if (allowedObjectDefinitionNames.contains(ALL_OBJECT_DEFINITIONS) ||
```
Comment (guilhermedcamacho) [inline modules/apps/object/object-api/src/main/java/com/liferay/object/scope/ObjectDefinitionsScoped.java:35]: IMHO this kind of condition looks strange. I would prefer to have something like
`if (ListUtil.isEmpty(allowedObjectDefinitionNames) || 
			allowedObjectDefinitionNames.contains(objectDefinitionName)) {
			return true;
		}`
Please see the comment in class [FunctionObjectActionExecutorImpl.java](https://github.com/liferay-objects/liferay-portal/pull/2355/files#diff-732c562833e89b50e882b03845dd7c6ede0b2521d4d3e6e0e10d1ff6832201fc)

File: modules/apps/object/object-api/src/main/java/com/liferay/object/scope/ObjectDefinitionsScoped.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+ */
+
+package com.liferay.object.scope;
+
+import java.util.List;
+
+/**
+ * @author Feliphe Marinho
+ */
+public interface ObjectDefinitionsScoped {
```
Comment (guilhermedcamacho) [inline modules/apps/object/object-api/src/main/java/com/liferay/object/scope/ObjectDefinitionsScoped.java:22]: Maybe the interface name should be in singular like `ObjectDefinitionScoped`, what do you think?

**Review by guilhermedcamacho — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/FunctionObjectActionExecutorImpl.java (hunk 1 above)
Comment (MarinhoFeliphe) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/FunctionObjectActionExecutorImpl.java]: Makes sense to me, thanks Camacho!

File: modules/apps/object/object-api/src/main/java/com/liferay/object/scope/ObjectDefinitionsScoped.java (hunk 2 above)
Comment (MarinhoFeliphe) [reply modules/apps/object/object-api/src/main/java/com/liferay/object/scope/ObjectDefinitionsScoped.java]: Agreed!

File: modules/apps/object/object-api/src/main/java/com/liferay/object/scope/ObjectDefinitionsScoped.java (hunk 3 above)
Comment (MarinhoFeliphe) [reply modules/apps/object/object-api/src/main/java/com/liferay/object/scope/ObjectDefinitionsScoped.java]: Agreed!

**Comment by MarinhoFeliphe** [comment]
Hey @guilhermedcamacho,

I've made the requested changes, could you please take a look again?

Thank you!

**Review by guilhermedcamacho — APPROVED** [review]

**Comment by guilhermedcamacho** [comment]
Hey @MarinhoFeliphe , please check if CI test errors are related.

**Comment by MarinhoFeliphe** [comment]
ci:forward:force

**Comment by MarinhoFeliphe** [comment]
Resent here: https://github.com/brianchandotcom/liferay-portal/pull/138146

**Pull request closed without merging**